Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/ne/InformationBlock.java

```diff
@@ -516,9 +515,15 @@ public String getTargetOpSysAsString() {
      */
     public String getApplicationFlagsAsString() {
         StringBuffer buffer = new StringBuffer();
-        if ((ne_flags_app & FLAGS_APP_FULL_SCREEN) != 0) {
+        byte application_type = ne_flags_app & 0x03;
```

Comment:
Are you not getting a compilation error here?

`Type mismatch: cannot convert from int to byte`

Reply:
I'm fixing it BTW, and formatting the entire file before pushing your changes.